### PR TITLE
feat: add polymorphic associations support

### PR DESCRIPTION
**Pull request by nicolasalexandre9**
### General

- [ ] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org)
- [ ] Test manually the implemented changes
- [ ] Validate the code quality (indentation, syntax, style, simplicity, readability)

### Security

- [ ] Consider the security impact of the changes made

### CODE REVIEW

File: spec/lib/forest_liana/schema_file_updater_spec.rb [hunk 1]
```diff
@@ -10,6 +10,84 @@ module ForestLiana
...
+                "actions" => [],
+                "segments" => []
+              }
+            ]
+            schema_file_updater = ForestLiana::SchemaFileUpdater.new("test.txt", collections, {})
+            expect(schema_file_updater.instance_variable_get(:@collections))
+              .to eq(collections)
+          end
+
+          describe "when a smart action field is malformed" do
```
Comment (matthv) [inline spec/lib/forest_liana/schema_file_updater_spec.rb]: seems not related to the PR

**Review by matthv — APPROVED** [review]

File: spec/lib/forest_liana/schema_file_updater_spec.rb (hunk 1 above)
Comment (nicolasalexandre9) [reply spec/lib/forest_liana/schema_file_updater_spec.rb]: 👍

Comment (matthv) [reply spec/lib/forest_liana/schema_file_updater_spec.rb]: outdated

**Comment by matthv** [comment]
Task linked: [CU-860t2z5wd Rails v1 - Add polymorphic support](https://app.clickup.com/t/860t2z5wd)

**Review by matthv — APPROVED** [review]
